pybwa/libbwaaln.pyi

```diff
@@ -55,5 +57,20 @@ class BwaAln:
     ) -> List[AlignedSegment]: ...
     def reinitialize_seed(self) -> None: ...
 
+@dataclass(frozen=True)
+class XaHit:
```

Comment:
Will you add a dosctring here or somewhere?  Specifically it would be good to know if `start` is exactly what's in the XA tag (1-based) or converted to zero based.

Reply:
I feel like I'd also be tempted to rename this to something like `AuxHit` because I think it's the exact same set of fields as an `SA` hit, and could foreseeably be reused there too.